src/nni_manager/training_service/kubernetes/kubeflow/kubeflowApiClient.ts

```diff
@@ -7,55 +7,9 @@ import * as fs from 'fs';
 import { GeneralK8sClient, KubernetesCRDClient } from '../kubernetesApiClient';
 import { KubeflowOperator } from './kubeflowConfig';
 
-/**
- * KubeflowOperator Client
- */
-abstract class KubeflowOperatorClient extends KubernetesCRDClient {
-    /**
-     * Factory method to generate operator client
-     */
-    // tslint:disable-next-line:function-name
-    public static generateOperatorClient(kubeflowOperator: KubeflowOperator,
-                                         operatorApiVersion: string): KubernetesCRDClient {
-        switch (kubeflowOperator) {
-            case 'tf-operator': {
-                switch (operatorApiVersion) {
-                    case 'v1alpha2': {
-                        return new TFOperatorClientV1Alpha2();
-                    }
-                    case 'v1beta1': {
-                        return new TFOperatorClientV1Beta1();
-                    }
-                    case 'v1beta2': {
-                        return new TFOperatorClientV1Beta2();
-                    }
-                    default:
-                        throw new Error(`Invalid tf-operator apiVersion ${operatorApiVersion}`);
-                }
-            }
-            case 'pytorch-operator': {
-                switch (operatorApiVersion) {
-                    case 'v1alpha2': {
-                        return new PyTorchOperatorClientV1Alpha2();
-                    }
-                    case 'v1beta1': {
-                        return new PyTorchOperatorClientV1Beta1();
-                    }
-                    case 'v1beta2': {
-                        return new PyTorchOperatorClientV1Beta2();
-                    }
-                    default:
-                        throw new Error(`Invalid pytorch-operator apiVersion ${operatorApiVersion}`);
-                }
-            }
-            default:
-                throw new Error(`Invalid operator ${kubeflowOperator}`);
-        }
-    }
-}
 
 // tslint:disable: no-unsafe-any no-any completed-docs
-class TFOperatorClientV1Alpha2 extends KubeflowOperatorClient {
```

Comment:
the previous version is a little strange, why extend `KubeflowOperatorClient` rather than `KubernetesCRDClient`? any considerations?

Reply:
It was a mistake.